### PR TITLE
fix: build ios and android 

### PR DESCRIPTION
**Pull request by anhle10051996**
- Fix build failed for staging and production scheme in ios
- Fix build failed when run app from android studio

### CODE REVIEW

**Comment by TranQuangPhi** [comment]
I don't think this PR is necessary, so would you be able to close it?

**Comment by baonguyenhsts** [comment]
> I don't think this PR is necessary, so would you be able to close it?

Please provide a reason for why we should close it.

File: template/android/app/build.gradle [hunk 1]
```diff
@@ -145,9 +145,24 @@ android {
...
+            applicationId project.env.get("APP_ID")
+            targetSdkVersion rootProject.ext.targetSdkVersion
+            resValue "string", "build_config_package", "com.saigontechnolgy.rnbaseproject"
+        }
+        production {
+            minSdkVersion rootProject.ext.minSdkVersion
+            applicationId project.env.get("APP_ID")
+            targetSdkVersion rootProject.ext.targetSdkVersion
+            resValue "string", "build_config_package", "com.saigontechnolgy.rnbaseproject"
+        }
```
Comment (baonguyenhsts) [inline template/android/app/build.gradle:165]: These fields have the same value. I recall that @loido removed this part previously.

File: template/src/constants/configs.js [hunk 2]
```diff
@@ -48,6 +48,19 @@ export const CODEPUSH_KEYS =
...
+        },
+
+        {
+          dev: '',
+          key: Platform.select({
+            android: '',
+            ios: '',
+          }),
+        },
+      ]
```
Comment (baonguyenhsts) [inline template/src/constants/configs.js:65]: Please add comments for these changes.

**Pull request closed without merging**